src/components/home/PanelResult.vue

```diff
@@ -56,28 +56,47 @@
           </template>
         </help-button>
       </v-card-title>
-      <v-card-text class="body-1">
+      <v-card-text>
         <info-alert :active="help">
           <template v-slot:content>
             <panel-results-help />
           </template>
         </info-alert>
-        <span v-if="userGenes.length > 0">
-          {{ $t('panel-result.result.title') }}: {{ userGenes.length }}
-        </span>
+        <v-container v-if="userGenes.length > 0 || notFoundGenes.length > 0">
+          <v-row no-gutters>
+            <v-col
+              cols="12"
+              sm="6"
+              md="4"
+              offset-md="4"
+            >
+              <v-simple-table class>
+                <template v-slot:default>
+                  <tbody>
+                    <tr v-if="userGenes.length > 0">
+                      <td>{{ $t('panel-result.result.title') }}</td>
+                      <td>{{ userGenes.length }}</td>
+                    </tr>
+                    <tr v-if="notFoundGenes.length > 0">
+                      <td>{{ $t('panel-result.result.not-found-genes-title') }}</td>
+                      <td>{{ notFoundGenes.length }}</td>
+                    </tr>
+                  </tbody>
+                </template>
+              </v-simple-table>
+            </v-col>
+          </v-row>
+        </v-container>
         <v-data-table
           :headers="tableHeaders"
-          :items="getPanelResult()"
+          :items="panelContent"
           item-key="name"
         >
           <template v-slot:[`item.institution`]="{ item }">
```

Comment:
Use a v-if here in case institution is null, otherwise there's a console error and an empty arrow

Reply:
should we do that?

Reply:
I don't understand your comment. Basically you need to make sure institution is not null or empty. v-if in the button will break the tooltip because it won't have a "v-on" anymore. The check needs to be at the v-tooltip level.

Reply:
done